frontend/src/components/AdminDash/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import styles from '../../styles/AdminDash.module.css';
+import ApplicantList from './ApplicantList';
+import ApplicantInfo from './AppInfo';
+import { Button, Input } from 'reactstrap';
+import Filters from './Filters';
+
+const dummyUsers = [
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  },
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  },
+  {
+    name: 'John Adams',
+    email: 'jadams76@foundingfathers.gov',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Henry Clay',
+    email: 'greatcompromiser@unelectable.edu',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Benjamin Franklin',
+    email: 'tobefrankwithyou@unsungheros.net',
+    role: 'Volunteer'
+  },
+  {
+    name: 'Alexander Hamilton',
+    email: 'bigbanks@dieyoung.com',
+    role: 'Volunteer'
+  },
+  {
+    name: 'James K. Polk',
+    email: 'napoleonofthestump@forgotten.gov',
+    role: 'Pending'
+  }
+];
+
+export default class AdminDash extends Component {
+  constructor() {
+    super();
+    this.state = {
+      selectedApplicantIndex: 0,
+      showFilterModal: false
+    };
+    this.onSelectApplicant = this.onSelectApplicant.bind(this);
+    this.onShowFilterModal = this.onShowFilterModal.bind(this);
+  }
+  onSelectApplicant(index) {
```

Comment:
This approach is gucci, but just for information, if you don't wanna do the explicit bind in the constructor you can do something like this
```js
onSelectApplicant = index => {...}
```

And then it'll automatically be bound and you'll be gucci :) But this is also a good way of doing it!

Reply:
Yeah I thought about that syntax but still did it the old way for some reason. I'll change it